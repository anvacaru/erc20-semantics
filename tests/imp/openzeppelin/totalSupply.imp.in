makeThrowAsPrintTrue;
makeAccount(1,100);
makeTotalSupply(100);
test.totalSupply();