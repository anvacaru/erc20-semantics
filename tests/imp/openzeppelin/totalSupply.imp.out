Total supply is 100
<IMP>
  <ERC20>
    <caller>
      0
    </caller>
    <k>
      .
    </k>
    <accounts>
      <account>
        <id>
          1
        </id>
        <balance>
          100
        </balance>
      </account>
    </accounts>
    <allowances>
      <allowance1>
        <owner>
          1
        </owner>
        <spenders>
          .AllowCellBag
        </spenders>
      </allowance1>
    </allowances>
    <log>
      Events:
    </log>
    <supply>
      100
    </supply>
  </ERC20>
  <state>
    .Map
  </state>
  <input>
    ListItem ( #buffer ( "" ) )
    ListItem ( "on" )
    ListItem ( #istream ( 0 ) )
  </input>
  <output>
    ListItem ( #ostream ( 1 ) )
    ListItem ( "on" )
    ListItem ( #buffer ( "" ) )
  </output>
  <throwAsPrint>
    true
  </throwAsPrint>
  <printStrings>
    true
  </printStrings>
</IMP>
